### ISSUE TITLE
Lyrics support 

### ISSUE THREAD

**Issue by tonyfung99** [jianpu/musje#19]
Will the format support lyrics display?

**Comment by malcomwu** [comment]
Yes, the lyrics support is in high priority... just I don't have time to do it.

**Comment by tonyfung99** [comment]
@malcomwu u may want to reference this.
https://github.com/felixhao28/react-jianpu
